### PR TITLE
fix(change): access api url from the user to hit APIs

### PR DESCRIPTION
**Pull request by arunkumars08**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -49,11 +49,33 @@ export class AppComponent implements OnInit {
...
-            console.log('=========================');
+            this.gateway['config'] = this.apiData['route_config'];
+
+            // {
+            //     "route_config": {
+            //         "api_url": ""
+            //     }
+            // }
+
+            // In case of 3 scale
```
Comment (invincibleJai) [inline src/app/app.component.ts]: we can remove comments

Comment (arunkumars08) [reply src/app/app.component.ts]: I kept it intentionally to handle for 3 scale.

File: src/app/stack/feedback/feedback.service.ts [hunk 2]
```diff
@@ -10,17 +10,20 @@ export class FeedbackService {
...
-                let headers: Headers = new Headers({'Content-Type': 'application/json'});
-                headers.append('Authorization', 'Bearer ' + params['access_token']);
-                return this.http
-                    .post(url, JSON.stringify(feedback), {
-                        headers: headers
-                    })
-                    .map(this.extractData)
-                    .catch(this.handleError);
+                if (params['config'] && params['config']['api_url']) {
+                    url = params['config']['api_url'] + url;
```
Comment (jyasveer) [inline src/app/stack/feedback/feedback.service.ts]: i think checking if the `/` is present here will be a nice thing. could you look at it ?

File: src/app/app.component.ts (hunk 1 above)
Comment (invincibleJai) [reply src/app/app.component.ts]: got it!!

File: src/app/stack/feedback/feedback.service.ts (hunk 2 above)
Comment (arunkumars08) [reply src/app/stack/feedback/feedback.service.ts]: That is handled in the app.component.ts itself.

Comment (arunkumars08) [reply src/app/stack/feedback/feedback.service.ts]: https://github.com/fabric8-analytics/fabric8-analytics-stack-report-ui/pull/41/files#diff-5c26d2c8f8838f32ace87fdecd6344eaR70

Comment (jyasveer) [reply src/app/stack/feedback/feedback.service.ts]: Okay. got it

**Pull request merged**